backend/adapter_processor/serializers.py

```diff
@@ -70,7 +72,20 @@ def to_representation(self, instance: AdapterInstance) -> dict[str, str]:
         adapter_metadata = json.loads(
             f.decrypt(bytes(instance.adapter_metadata_b).decode("utf-8"))
         )
+        # Get the adapter_instance
+        adapter_class = Adapterkit().get_adapter_class_by_adapter_id(
+            instance.adapter_id
+        )
+        adapter_instance = adapter_class(adapter_metadata)
+        # If adapter_instance is a LLM send
+        # additional parameter of context_window
+        if isinstance(adapter_instance, LLMAdapter):
+            adapter_metadata["context_window_size"] = (
```

Comment:
Is it to show in UI?

Reply:
Not directly. A calculation of this will show up in UI.